charts/cluster-api-runtime-extensions-nutanix/templates/helm-config.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+data:
+  aws-ccm: |
+    defaultChartName: aws-cloud-controller-manager
```

Comment:
nit: `default` prefix to all the fields may be not necessary. It would be good to use the generic names `chartName` `chartVersion`, `repositoryURL`. since the name of the configmap is provided as input to the cli with default to `default-helm-addons-config` and the code does not need to be aware if default config is used or not, removing `default` prefix would be better.

Reply:
➕